packages/expo-video/src/VideoView.web.tsx

```diff
@@ -94,6 +94,20 @@ export const VideoView = forwardRef((props: { player?: VideoPlayer } & VideoView
     hasToSetupAudioContext.current = false;
   }
 
+  function maybeSetupFullscreenListener(element: HTMLVideoElement) {
+    if (!props.allowsFullscreen) {
```

Comment:
Have you tested if that works fine when `allowsFullscreen` is initially set to `false`, but then is changed to `true`. Will the event listener be added in that case?

Reply:
@tsapeta Hey, yes, you're correct, it doesn't fire in that case. I went ahead and removed this guard so that its registered. My initial motivation was to "avoid" unnecessarily registering the fullscreen listener if fullscreen is not allowed, but on second thought, maybe its a bit overengineered if we do this. Also, by removing this guard, the behavior is now aligned with the native platforms where the event always fires, no matter the state of the `allowsFullscreen` prop. I would argue that its only worth adjusting this behavior if it proves to be problematic in the future.